packages/metascraper-pdf/test/fixtures/download.sh

```
@@ -0,0 +1 @@
+nl -ba urls.txt | xargs -n 2 -P 8 sh -c 'curl -fL --retry 3 --retry-all-errors -o "$0.pdf" "$1"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)